packages/@lwc/synthetic-shadow/scripts/rollup/rollup.config.js

```diff
@@ -39,13 +39,14 @@ function rollupConfig({ wrap } = {}) {
             nodeResolve({
                 only: [/^@lwc\//],
             }),
-            rollupTypescript({
-                target: 'es2017',
-                tsconfig: path.join(__dirname, '../../tsconfig.json'),
-                noEmitOnError: true,
-            }),
+            typescript(),
             lwcFeatures(),
         ].filter(Boolean),
+        onwarn({ code, message }) {
+            if (!process.env.ROLLUP_WATCH && code !== 'CIRCULAR_DEPENDENCY') {
+                throw new Error(message);
+            }
+        },
```

Comment:
We already do this `onwarn` check for `CIRCULAR_DEPENDENCY` elsewhere. I wanted to get rid of the console warning for this package too.